spec/services/processors/code_climate_updater_spec.rb

```diff
@@ -0,0 +1,15 @@
+require 'rails_helper'
+
+describe Processors::CodeClimateUpdater do
+  describe '#call' do
+    let!(:first_project) { create :project }
+    let!(:second_project) { create :project }
+    let(:subject) { described_class.call }
+
+    it('calls UpdateProjectService on each project') do
+      expect(CodeClimate::Link).to receive(:call).once.with(first_project)
+      expect(CodeClimate::Link).to receive(:call).once.with(second_project)
```

Comment:
I'd rather use Webmock so the test is actually testing a request to codeclimate

Reply:
https://github.com/bblimke/webmock#setting-expectations-in-rspec-on-webmock-module